src/cwl_platform/omics_platform.py

```diff
@@ -0,0 +1,358 @@
+        '''
+        raise ValueError("Omics does not support get_project. Use get_project_by_id or get_project_by_name instead.")
+
+    def get_project_by_name(self, project_name):
+        ''' Return a dictionary of project to provide project_name and project_id tag info for omics jobs '''
+        response = self.api.list_run_groups(
+            name=project_name, maxResults=100
+        )
+        if len(response['items'])>0:
+            run_group_id = response['items'][0]['id']
+
+            project = {
+                'ProjectId': run_group_id
+            }
+            return project
+        else:
+            logger.error('Could not find project with name: %s', project_name)
+            return {}
+
+    def get_project_by_id(self, project_id):
+        ''' Return a dictionary of project to provide project_id tag info for omics jobs'''
+        project = {
+            'ProjectId': project_id
+        }
+        return project
+
+    def get_project_cost(self, project):
+        ''' Return project cost '''
+        # TODO: Return total cost from run_group_id
+        return 0
+
+    def get_project_users(self, project):
+        ''' Return a list of user objects associated with a project '''
+        ''' TODO '''
+        return None
+
+    def get_projects(self):
+        ''' Get list of all projects '''
+        # TODO: I think this should return a list of project names using run group id
+        return None
+
+    def add_user_to_project(self, platform_user, project, permission):
+        """ 
+        Add a user to a project on the platform
+        :param platform_user: platform user (from get_user)
+        :param project: platform project
+        :param permission: permission (permission="read|write|execute|admin")
+        """
+        ''' TODO '''
+        return
+
+    def get_user(self, user):
+        '''Get a user object from their (platform) user id or email address'''
+        raise ValueError("Not yet implemented")
+
+    def rename_file(self, fileid, new_filename):
+        raise ValueError("Not yet implemented")
+
+    def roll_file(self, project, file_name):
+        raise ValueError("Not yet implemented")
+
+    def stage_output_files(self, project, output_files):
+        ''' TODO '''
+        return
+
+    def stage_task_output(self, task, project, output_to_export, output_directory_name):
+        ''' TODO '''
+        return
+
+    @retry(wait=wait_fixed(1), stop=stop_after_attempt(3))
+    def submit_task(self, name, project, workflow, parameters, execution_settings=None):
+        '''
+        Submit workflow for one sample.
+        name: sample ID.
+        project: dictionary of {'ProjectName': 'string'} or {'ProjectId': 'string'}
+        workflow: workflow ID in omics.
+        parameters: dictionary of input parameters.
+
+        return omics response for start_run.
+        '''
+        base_output_path = f"s3://{self.output_bucket}/Project/"
```

Comment:
I don't like that we could have two different output locations.  We should be consistent about this.  ProjectId is actually runGroupId I believe.  Can we update this key to be runGroupId so its meaning is clear?